liquidprompt

```diff
@@ -1867,19 +1867,25 @@ _lp_http_proxy_color() {
     lp_http_proxy_color="${LP_COLOR_PROXY}${LP_MARK_PROXY}${NO_COL}"
 }
 
-_lp_env_vars() { # [color_set, [color_unset]]
+_lp_env_vars() { # [color_set, [color_unset, [end_color]]]
     # Expects LP_ENV_VARS to be an array containing items of the form:
     # "<ENV_VAR_NAME> <string if set>[ <string if unset>]"
     # Strings may be `%s`, which will be replaced by the variable's actual content.
+    # <end_color> is added after each variable.
+    # If end_color is not passed, it will default to $NO_COL,
+    # if it is " ", it will be set to an empty string.
 
     (( LP_ENABLE_ENV_VARS )) || return 2
 
     local color_set="${1-}"
     local color_unset="${2-}"
-    local color_no="${NO_COL}"
+    local color_no="${3:-$NO_COL}"
```

Comment:
Why not use 
```suggestion
    local color_no="${3-$NO_COL}"
```
instead, which would let a user pass in the empty string if that's what they want.

Then you don't need to add the below block. And you don't need to explain the strange behavior in the docs.

Reply:
Of course, should have seen it 🤦